util/buildRelease/install.sh

```diff
@@ -347,17 +346,26 @@ then
   fi
 fi
 
+# create symlink for chpldoc-legacy
```

Comment:
This comment is redundant with the one at the start of the `then` branch of the below `if`, one of them should go